### PR TITLE
Compatible with Codec & Wasm32

### PR DESCRIPTION
**Pull request by yrong**

### CODE REVIEW

File: src/keys.rs [hunk 1]
```diff
@@ -112,8 +115,7 @@ impl Drop for SecretKey {
 }
 
 /// A BLS public key.
-#[derive(Clone, PartialEq, Eq)]
-#[cfg_attr(feature = "std", derive(Debug))]
+#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, Encode, Decode, TypeInfo, MaxEncodedLen)]
```
Comment (vgeddes) [inline src/keys.rs]: Should derive RuntimeDebug instead of Debug to reduce code size in wasm.

Comment (yrong) [reply src/keys.rs]: https://github.com/Snowfork/milagro_bls/pull/3/commits/a8e9585ae904112328ddff9c229db473b563f4fa

Comment (vgeddes) [reply src/keys.rs]: Sorry can we use `#[cfg_attr(feature = "std", derive(Debug))]` instead of RuntimeDebug, to not rely on substrate.

**Comment by vgeddes** [comment]
Can you fix the merge conflicts please.

**Comment by vgeddes** [comment]
obsolete. Applied these changes to the `snowbridge` branch

**Pull request closed without merging**